scripts/core/editor3/actions/spellchecker.jsx

```diff
@@ -0,0 +1,13 @@
+export function replaceWord(word) {
+    return {
+        type: 'REPLACE_WORD',
+        payload: word
+    };
+}
+
+export function showContextMenu(contextMenuData) {
+    return {
+        type: 'SHOW_CONTEXT_MENU',
```

Comment:
Here it would make sense too, because we might have other context menus in the future: `SPELLCHECKER_CONTEXT_MENU` ?